lib/worker.js

```diff
@@ -36,6 +36,13 @@ app.use(express.responseTime());
 app.use(express.static(public_dir, {
   maxAge: 1000 * 60 * 1
 }));
+
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
```

Comment:
Should we be allowing `*` or be more selective like we are for our S3 CORS policies? @tayseph